### DISCUSSION TITLE
Mountain Duck Windows 11 crash

### DISCUSSION THREAD

**Discussion by IT-Gipsy** [iterate-ch/cyberduck#14396] [Q&A]
Good day,

after the installation of MountainDuck, on my Windows Desktop PC (OS build 22621.1265)
my system keeps crashing. This happened first after locking my PC (Windows Key + L) and then after restarting (as soon as Mountain Duck tries to establish a new drive letter with Amazon S3)

After de-installation of Mountain Duck, the issue was gone (my system runs stable again)
Mountain Duck is way more user friendly then Cyber Duck. As it "maps" a network drive.
I would like to use Mountain Duck (and pay for the usage) if it doesn't crash my PC

Are there any LOG files I can find to send ?
Thanks, Harpert

**Comment by AliveDevil** [comment]
For summary purposes:
- Install Mountain Duck, and restart (prompted by Installer)
- Login, create a network drive, and work with that
- Lock the screen, restart Computer
- Login, system crashes, because Mountain Duck created a network drive

Can you send us matching entries from the Reliability Report of Windows?
